doc/nrf/ug_zigbee_qsg.rst

```diff
@@ -199,6 +203,17 @@ To program the samples, complete the following steps:
    On the development kit programmed with the light bulb sample, **LED 3** turns on when the light bulb joins the network.
    On the development kit programmed with the light switch sample, **LED 3** turns on when the device joins the network and **LED 4** turns on when the light switch finds a light bulb to control.
 
+.. note::
+
+   If you find that you are having issues with your devices joining the Zigbee Network, it may be because the connection needs to be reopened manually.
+   If the network is open, then **LED 3** on the developement kit programmed with the network coordinator sample will be on.
+   If **LED 3** is not on, then the network is closed and needs to be reopened again.
+
+   To do this, press **Button 1** on the developement kit that has been programmed as the network coordinator.
```

Comment:
```suggestion
   To reopen the network, press **Button 1** on the developement kit that has been programmed as the network coordinator.
```
Just a nit, to avoid the deictic reference that might be a bit (just a bit) confusing.